packages/notification-center/src/hooks/useInitializeSocket.ts

```diff
@@ -11,6 +12,7 @@ type IUseInitializeSocket = (args: { socketUrl: string }) => {
 
 export const useInitializeSocket: IUseInitializeSocket = ({ socketUrl }) => {
   const socketRef = useRef<Socket | null>(null);
+  const setQueryKey = useSetQueryKey();
```

Comment:
Hi @gitstart I believe your problem is coming from here. `useInitializeSocket` is used in NovuProvider.ts but `useSetQueryKey` uses the context of that provider.

Reply:
I have effected the changes @ainouzgali